### PR TITLE
Fix typos in tutorials

### PR DESCRIPTION
**Pull request by LaRiffle**

### CODE REVIEW

File: examples/Part 1 - Launch a Grid Node Locally.ipynb [hunk 1]
```diff
@@ -44,7 +44,8 @@
     "\n",
     "### Step 3: Start App\n",
     "\n",
-    "Then to start the app just run the `websocket_app` script.\n",
+    "Then, to start the app, just run the `start_websocket` script.\n",
```
Comment (midokura-silvia) [inline examples/Part 1 - Launch a Grid Node Locally.ipynb]: remove line (artifact of merge)

Comment (LaRiffle) [reply examples/Part 1 - Launch a Grid Node Locally.ipynb]: done

Comment (mari-linhares) [reply examples/Part 1 - Launch a Grid Node Locally.ipynb]: thanks

**Comment by mari-linhares** [comment]
I'm working on some other fixes using this branch to speed up merging tutorial fixes.

**Pull request merged**